cli/src/main.rs

```diff
@@ -23,10 +25,16 @@ enum Command {
     CreateOrganization(create_organization::CreateOrganization),
     /// List all devices
     ListDevices(list_devices::ListDevices),
+    #[cfg(feature = "testenv")]
+    /// Create a temporary environment and initialize a test setup for parsec.
+    /// #### WARNING: it also leaves an in-memory backend running in the background on port `6770`.
```

Comment:
I didn't see a cleanup for the in-memory server, that's expected ?

Reply:
It is closed with your terminal